pudb/run.py

```diff
@@ -27,6 +33,7 @@ def get_argparse_parser():
         sys.exit(os.EX_OK)
 
     parser = argparse.ArgumentParser(
+        'pudb3',
```

Comment:
I notice `python -m pudb --print-completion zsh` will generate `#compdef __main__`. This line can make it always `#compdef pudb3`